scrapinghub/client/activity.py

```diff
@@ -10,51 +10,51 @@ class Activity(_Proxy):
     Not a public constructor: use :class:`Project` instance to get a
     :class:`Activity` instance. See :attr:`Project.activity` attribute.
 
-    Please note that list() method can use a lot of memory and for a large
```

Comment:
The lines are moved to `_Proxy.list` method to avoid copy-pasting it everywhere.